internal/cmd/jsonschema-generate-testdata/main.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2023 Buf Technologies, Inc.
+// Copyright 2024 Buf Technologies, Inc.
```

Comment:
Copyright years should be additive, so this (and everything else) should be `2023-2024`.